buildtest/cli/report.py

```diff
@@ -552,6 +560,7 @@ def print_report(
             root_disk_usage|PASS|0
             root_disk_usage|PASS|0
         """
+        consoleColor = checkColor(color) or "blue"
```

Comment:
you can remove the `or "blue"`. The method `checkColor` will return the default anyways

Reply:
removed